setuptools_rust/setuptools_ext.py

```diff
@@ -287,3 +279,12 @@ def rust_extensions(
     if has_rust_extensions:
         patch_distutils_build()
         add_rust_extension(dist)
+
+
+_CARGO_VENDOR_CONFIG = b"""
+[source.crates-io]
+replace-with = "vendored-sources"
+
+[source.vendored-sources]
+directory = ".cargo/vendor"
+"""
```

Comment:
@messense the windows CI was failing, I think the number of backslashes changed or something awkward like that. I figured might as well just hard-code it like this. Sound reasonable to you?

Reply:
Sounds reasonable.